internal/service/anycast/api_anycast_config_data_source_test.go

```diff
@@ -37,7 +37,7 @@ func TestAccOnPremAnycastManagerDataSource_Services(t *testing.T) {
 }
 
 func TestAccOnPremAnycastManagerDataSource_IsConfigured(t *testing.T) {
```

Comment:
All test names in this file should also be renamed -- 
```suggestion
func TestAccAnycastConfigDataSource_IsConfigured(t *testing.T) {
```

Reply:
done